database/src/main/resources/changesets/metadata_typed_values.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="tjeandet" id="metadata_typed_values">
+        <addColumn
+                tableName="METADATA_JOURNAL">
+            <column name="METADATA_VALUE_TYPE" type="VARCHAR(255)" />
+        </addColumn>
+
+        <modifySql dbms="mysql">
+            <append value=" ENGINE=INNODB"/>
+        </modifySql>
```

Comment:
I just copied this from earlier Liquibase work which was probably motivated by the references Khalid links above.